man/texreg.Rd

```diff
@@ -90,6 +90,15 @@ identical.
 Sometimes it happens that the same variable has a different name in different models. In this case, the user can use this function to assign identical names. If possible, the rows will then be merged into a single row unless both rows contain values in the same column.
 
 Where the argument contains an \code{NA} value, the original name of the coefficient is kept. For example, \code{custom.coef.names = c(NA, "age", NA)} will only replace the second coef name and leave the first and third name as they are in the original model.}
+\item{custom.coef.map}{The \code{custom.coef.map} argument can be used to select, omit, rename, and reorder coefficients.
+    
+Users must supply a named list of this form: \code{list('x'='First variable', 'y'=NA, 'z'='Third variable')}. With that particular example of \code{custom.coef.map}, 
```

Comment:
Can you put spaces around `=`? Sorry for nitpicking, but the Editors of the Journal of Statistical Software educated me in this regard...